### ISSUE TITLE
`cargo build` does not work anymore

### ISSUE THREAD

**Issue by emilengler** [rosenpass/rosenpass#96]
Since #90 got merged, it is not possible to run `cargo build` anymore.
It will fail with the following:
```
   Compiling clap v4.3.0
   Compiling oqs-sys v0.7.2
error: failed to run custom build command for `oqs-sys v0.7.2`

Caused by:
  process didn't exit successfully: `/home/engl_em/rosenpass/target/debug/build/oqs-sys-8108111108712533/build-script-build` (exit status: 101)
  --- stdout
  CMAKE_TOOLCHAIN_FILE_x86_64-unknown-linux-gnu = None
  CMAKE_TOOLCHAIN_FILE_x86_64_unknown_linux_gnu = None
  HOST_CMAKE_TOOLCHAIN_FILE = None
  CMAKE_TOOLCHAIN_FILE = None
  CMAKE_GENERATOR_x86_64-unknown-linux-gnu = None
  CMAKE_GENERATOR_x86_64_unknown_linux_gnu = None
  HOST_CMAKE_GENERATOR = None
  CMAKE_GENERATOR = None
  CMAKE_PREFIX_PATH_x86_64-unknown-linux-gnu = None
  CMAKE_PREFIX_PATH_x86_64_unknown_linux_gnu = None
  HOST_CMAKE_PREFIX_PATH = None
  CMAKE_PREFIX_PATH = Some("/nix/store/fl200crqmphv7jxd2kc34bld7gqwhvkl-cargo-release-0.24.10:/nix/store/f5rx5wh910w5mdhjbf933g2n47ldx1dw-clippy-1.69.0:/nix/store/f2na5pxm3gw45db6przrb66sgnyxyjps-prettier-2.8.8:/nix/store/syra5gb6m72riy41ky1vzsr2nlmh3jkq-rustfmt-1.69.0:/nix/store/jvi8qhmxlqw6b95m06cav9r6c40n4faj-proverif-2.04:/nix/store/qjdm6kng0l48d2dg2xv65hqarm2lz8wj-cmake:/nix/store/i7z0045pwjqwf2qlxnvyd0ga97pklrfb-rust-mixed:/nix/store/by92xjwrf1nrlxnrvrpjpgb5hag41k4y-jq-1.6-dev:/nix/store/s9j08pc615ii5z8rln2yjarsc9zf7q04-jq-1.6-bin:/nix/store/fr6j11238dbxmpn1yxqg4i8ipfvmc704-jq-1.6-lib:/nix/store/jzhlbzh5w9zibvilrh6pv9lf2d1iz55m-rsync-3.2.7:/nix/store/7wkshj58fcsl1f3zyi67qsxgl1p8nki1-gcc-wrapper-12.2.0:/nix/store/rbwalnmk5a6nj9mbldxzmj11gvacbrsi-binutils-wrapper-2.40:/nix/store/fic9avx8yabwb3ib8m49ld3kq9fkwwwd-cmake-3.25.3:/nix/store/a3pyfxzvwhmwccfxv69hinzrglxq297w-mandoc-1.14.6:/nix/store/j7ch0sr0mvf94qyrb2r94q4hmbbrzvlj-make-shell-wrapper-hook:/nix/store/wj5zmkpghcbfqs6rqnr17vm92bx6fzpi-die-hook:/nix/store/09jbh8y3jbq0gs47793rzn6i7yaabxmp-pkg-config-wrapper-0.29.2:/nix/store/md18nky9dgn0qg2lg460l65v6s96q8sm-remove-references-to:/nix/store/l1mfby7i4qd5mprzidc4ibcc89mg7y5x-rust-bindgen-hook:/nix/store/ywwjpdyhar4f3vcqf4qk77vrbr3vj5wl-patchelf-0.15.0:/nix/store/mqjmlpc8xxkgmyhxmlf64sms1jscw608-bash-5.2-p15-dev:/nix/store/rhvbjmcfnkg8i2dxpzr114cp1ws7f667-bash-5.2-p15:/nix/store/fydlpvccndfjd23ma8xagy6765cd8ffb-libsodium-1.0.18-dev:/nix/store/26fp2r4q9s8wbp96jkc801m6paglgqkd-libsodium-1.0.18")
  CMAKE_x86_64-unknown-linux-gnu = None
  CMAKE_x86_64_unknown_linux_gnu = None
  HOST_CMAKE = None
  CMAKE = None
  running: cd "/home/engl_em/rosenpass/target/debug/build/oqs-sys-306f8554864eb54f/out/build" && CMAKE_PREFIX_PATH="/nix/store/fl200crqmphv7jxd2kc34bld7gqwhvkl-cargo-release-0.24.10:/nix/store/f5rx5wh910w5mdhjbf933g2n47ldx1dw-clippy-1.69.0:/nix/store/f2na5pxm3gw45db6przrb66sgnyxyjps-prettier-2.8.8:/nix/store/syra5gb6m72riy41ky1vzsr2nlmh3jkq-rustfmt-1.69.0:/nix/store/jvi8qhmxlqw6b95m06cav9r6c40n4faj-proverif-2.04:/nix/store/qjdm6kng0l48d2dg2xv65hqarm2lz8wj-cmake:/nix/store/i7z0045pwjqwf2qlxnvyd0ga97pklrfb-rust-mixed:/nix/store/by92xjwrf1nrlxnrvrpjpgb5hag41k4y-jq-1.6-dev:/nix/store/s9j08pc615ii5z8rln2yjarsc9zf7q04-jq-1.6-bin:/nix/store/fr6j11238dbxmpn1yxqg4i8ipfvmc704-jq-1.6-lib:/nix/store/jzhlbzh5w9zibvilrh6pv9lf2d1iz55m-rsync-3.2.7:/nix/store/7wkshj58fcsl1f3zyi67qsxgl1p8nki1-gcc-wrapper-12.2.0:/nix/store/rbwalnmk5a6nj9mbldxzmj11gvacbrsi-binutils-wrapper-2.40:/nix/store/fic9avx8yabwb3ib8m49ld3kq9fkwwwd-cmake-3.25.3:/nix/store/a3pyfxzvwhmwccfxv69hinzrglxq297w-mandoc-1.14.6:/nix/store/j7ch0sr0mvf94qyrb2r94q4hmbbrzvlj-make-shell-wrapper-hook:/nix/store/wj5zmkpghcbfqs6rqnr17vm92bx6fzpi-die-hook:/nix/store/09jbh8y3jbq0gs47793rzn6i7yaabxmp-pkg-config-wrapper-0.29.2:/nix/store/md18nky9dgn0qg2lg460l65v6s96q8sm-remove-references-to:/nix/store/l1mfby7i4qd5mprzidc4ibcc89mg7y5x-rust-bindgen-hook:/nix/store/ywwjpdyhar4f3vcqf4qk77vrbr3vj5wl-patchelf-0.15.0:/nix/store/mqjmlpc8xxkgmyhxmlf64sms1jscw608-bash-5.2-p15-dev:/nix/store/rhvbjmcfnkg8i2dxpzr114cp1ws7f667-bash-5.2-p15:/nix/store/fydlpvccndfjd23ma8xagy6765cd8ffb-libsodium-1.0.18-dev:/nix/store/26fp2r4q9s8wbp96jkc801m6paglgqkd-libsodium-1.0.18" "cmake" "/home/engl_em/.cargo/registry/src/index.crates.io-6f17d22bba15001f/oqs-sys-0.7.2/liboqs" "-DOQS_BUILD_ONLY_LIB=Yes" "-DOQS_DIST_BUILD=Yes" "-DOQS_ENABLE_KEM_BIKE=No" "-DOQS_ENABLE_KEM_CLASSIC_MCELIECE=Yes" "-DOQS_ENABLE_KEM_FRODOKEM=No" "-DOQS_ENABLE_KEM_HQC=No" "-DOQS_ENABLE_KEM_KYBER=Yes" "-DOQS_ENABLE_KEM_NTRU=No" "-DOQS_ENABLE_KEM_NTRUPRIME=No" "-DOQS_ENABLE_KEM_SABER=No" "-DOQS_ENABLE_KEM_SIDH=No" "-DOQS_ENABLE_KEM_SIKE=No" "-DOQS_ENABLE_SIG_DILITHIUM=No" "-DOQS_ENABLE_SIG_FALCON=No" "-DOQS_ENABLE_SIG_PICNIC=No" "-DOQS_ENABLE_SIG_RAINBOW=No" "-DOQS_ENABLE_SIG_SPHINCS=No" "-DOQS_USE_OPENSSL=No" "-DCMAKE_INSTALL_PREFIX=/home/engl_em/rosenpass/target/debug/build/oqs-sys-306f8554864eb54f/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_C_COMPILER=/nix/store/7wkshj58fcsl1f3zyi67qsxgl1p8nki1-gcc-wrapper-12.2.0/bin/gcc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_CXX_COMPILER=/nix/store/7wkshj58fcsl1f3zyi67qsxgl1p8nki1-gcc-wrapper-12.2.0/bin/g++" "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_ASM_COMPILER=/nix/store/7wkshj58fcsl1f3zyi67qsxgl1p8nki1-gcc-wrapper-12.2.0/bin/gcc" "-DCMAKE_BUILD_TYPE=Release"
  running: cd "/home/engl_em/rosenpass/target/debug/build/oqs-sys-306f8554864eb54f/out/build" && "cmake" "--build" "." "--target" "oqs" "--config" "Release" "--parallel" "64"
  cargo:root=/home/engl_em/rosenpass/target/debug/build/oqs-sys-306f8554864eb54f/out
  cargo:rustc-link-lib=static=oqs
  cargo:rustc-link-search=native=/home/engl_em/rosenpass/target/debug/build/oqs-sys-306f8554864eb54f/out/build/lib

  --- stderr
  thread 'main' panicked at 'Unable to generate bindings: NotExist("/home/engl_em/rosenpass/target/debug/build/oqs-sys-306f8554864eb54f/out/build/include/oqs/common.h")', /home/engl_em/.cargo/registry/src/index.crates.io-6f17d22bba15001f/oqs-sys-0.7.2/build.rs:33:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

**Comment by emilengler** [comment]
cc @wucke13

**Comment by wucke13** [comment]
fixed by #97
